examples/imageviewer/shared/src/commonMain/kotlin/example/imageviewer/view/ScalableImage.common.kt

```diff
@@ -55,22 +44,22 @@ internal fun ScalableImage(scalableState: ScalableState, image: ImageBitmap, mod
                 .pointerInput(Unit) {
                     detectTransformGestures { centroid, pan, zoom, _ ->
                         scalableState.addPan(pan)
-                        scalableState.addScale(zoom, centroid - areaCenter)
+                        scalableState.addZoom(zoom, centroid - areaCenter)
                     }
                 }
                 .onPointerEvent(PointerEventType.Scroll) {
                     val centroid = it.changes[0].position
                     val delta = it.changes[0].scrollDelta
                     val zoom = 1.2f.pow(-delta.y)
-                    scalableState.addScale(zoom, centroid - areaCenter)
+                    scalableState.addZoom(zoom, centroid - areaCenter)
                 }
                 .pointerInput(Unit) {
                     detectTapGestures(onDoubleTap = { position ->
-                        scalableState.setScale(
-                            if (scalableState.scale > 2.0) {
-                                scalableState.scaleLimits.start
+                        scalableState.setZoom(
+                            if (scalableState.zoom > 1.5f) {
```

Comment:
It's not super clear what exactly this check do and why `1.5`?

Reply:
Added a comment and extracted a constant